src/mdast-process-images.js

```diff
@@ -21,14 +21,43 @@ export class TooManyImagesError extends Error {
  * @param {object} tree
  * @param {MediaHandler} mediaHandler
  * @param {string} baseUrl
+ * @param {Array<string>} externalImagesUrlPrefixes Array of url prefixes to detect external images
  */
-export async function processImages(log, tree, mediaHandler, baseUrl) {
+export async function processImages(
+  log,
+  tree,
+  mediaHandler,
+  baseUrl,
+  externalImagesUrlPrefixes = [],
+) {
   if (!mediaHandler) {
     return;
   }
   // gather all image nodes
   const images = new Map();
+  const externalImageNodes = new Map();
```

Comment:
why do you track the external images, when you don't use the collection afterwards?

Reply:
i think its a remnant of my earlier change wherein I was counting the external images for the 200 limit. Since we no longer do that, no need for this tracking as well. Removing.